idl/github.com/uber/tchannel/meta.thrift

```diff
@@ -1,6 +1,23 @@
+enum State {
+    REFUSING = 0,
+    ACCEPTING = 1,
+    STOPPING = 2,
+    STOPPED = 3,
+}
+
+enum RequestType {
```

Comment:
Yep, added some comments here. I actually renamed it to `PROCESS` instead of `LEGACY` which doesn't really provide any useful information.